packages/webapp/pages/squads/[handle]/[token].tsx

```diff
@@ -74,10 +76,17 @@ const SquadReferral = ({ token, handle }: SquadReferralProps): ReactElement => {
     },
   );
   const sourceId = member?.source?.id;
-  const squadsUrl = `/squads/${sourceId}`;
   const { mutateAsync: onJoinSquad } = useMutation(
     () => joinSquadInvitation({ sourceId, token }),
-    { onSuccess: () => router.replace(squadsUrl) },
+    {
```

Comment:
Can we also handle the error here? What if the token is invalid?

Reply:
I tried this link https://preview2.app.daily.dev/squads/google_club/faketoken I didnt see any errors?

Reply:
Oh. Let me check that one then. Thanks for the quick link.

Reply:
I have updated the function call to fetch the token details and it is now handled correctly. Good catch!